docs/developers/account-storage.md

```diff
@@ -0,0 +1,55 @@
+---
+id: account-storage
+title: Account storage
+---
+
+## Description
+
+The Elrond protocol offers the possibility of storing additional data under an account as key-value pairs. This can be useful for many use cases.
+
+A wallet owner can store key-value pairs by using the built-in function `SaveKeyValue` that receives any number of key-value pairs.
+
+:::tip
+Keys that begin with `ELROND` will be rejected because they are reserved for protocol usage.
+:::
+
+## Transaction format
+
+```
+SaveKeyValueTransaction {
+    Sender: <account address of the wallet owner>
+    Receiver: <same as sender>
+    Value: 0
+    GasLimit: 250000 + additional gas limit*
+    Data: "SaveKeyValue" +
+          "@" + <key in hexadecimal encoding> +
+          "@" + <value in hexadecimal encoding> +
+          "@" + <key in hexadecimal encoding> +
+          "@" + <value in hexadecimal encoding> +
+          ...
+}
+```
+
+*The gas used for saving a pair is 250,000. However, this will be higher because of the length of the data field or the storage saving. 
```

Comment:
Replace 
However, this will be higher because of the length of the data field or the storage saving. 
with
A storage fee of 0.00x EGLD per character will also be incurred.

Please add the correct cost per byte

Reply:
done. added a complete explanation of the gas units consumed